Packs/DigitalGuardian/Integrations/DigitalGuardianARCEventCollector/DigitalGuardianARCEventCollector.py

```diff
@@ -263,31 +272,16 @@ def main() -> None:  # pragma: no cover
             return_results(result)
 
         elif command == 'digital-guardian-get-events':
+            events, last_run, command_results = get_events_command(client, args)
+            return_results(command_results)
+
             should_push_events = argToBoolean(args.pop('should_push_events'))
-            events, results = get_events_command(client, args)
-            return_results(results)
             if should_push_events:
-                add_time_to_events(events)
-                send_events_to_xsiam(
-                    events,
-                    vendor=VENDOR,
-                    product=PRODUCT
-                )
+                push_and_set_last_run(client, events, last_run)
+
         elif command == 'fetch-events':
-            last_run = demisto.getLastRun()
-            next_run, events = fetch_events(
-                client=client,
-                last_run=last_run,
-                limit=limit
-            )
-            add_time_to_events(events)
-            send_events_to_xsiam(
-                events,
-                vendor=VENDOR,
-                product=PRODUCT
-            )
-            # saves next_run for the time fetch-events is invoked
-            demisto.setLastRun(next_run)
+            events, last_run = fetch_events(client)  # type: ignore
```

Comment:
why `type:ignore`?

Reply:
Removed `#type: ignore` in commit ef814676d12876d5adc3cde92e5f7ff92c595e24